common/src/main/java/io/netty/util/ResourceLeakDetector.java

```diff
@@ -119,7 +121,7 @@ public static void setEnabled(boolean enabled) {
      * Returns {@code true} if resource leak detection is enabled.
      */
     public static boolean isEnabled() {
-        return getLevel().ordinal() > Level.DISABLED.ordinal();
```

Comment:
I would be interested to know what profiling results show if `level` is made `final`. Also I'm not sure `ENABLED` can be final if `level` is not `final`...and if this is the case I wonder if it is worth adding another variable at all?  Can we verify these assumptions with a benchmark?